homeassistant/components/rollershutter/zwave.py

```diff
@@ -40,29 +40,47 @@ class ZwaveRollershutter(zwave.ZWaveDeviceEntity, RollershutterDevice):
 
     def __init__(self, value):
         """Initialize the zwave rollershutter."""
+        import libopenzwave
         from openzwave.network import ZWaveNetwork
         from pydispatch import dispatcher
         ZWaveDeviceEntity.__init__(self, value, DOMAIN)
+        self._lozwmgr = libopenzwave.manager
```

Comment:
Got `AttributeError: 'module' object has no attribute 'manager'` here.

Should this be `libopenzwave.PyManager()`?